### DISCUSSION TITLE
Wrappers For other Frameworks/Languages

### DISCUSSION THREAD

**Discussion by doodirock** [SuryaSekhar14/vercel_blob#3] [General]
I stumbled upon your wrapper and figured you might be the best person to ask this too.  I recently went down the road of working with Vercel Blog outside a Nextjs app within ASP and figured it would be as simple as reading through their API docs.  Is it just me, or is their no clear documentation about how to use blog outside of Nextjs and no direct API?  I assume you reversed engineered the SDK to get your info to build the wrapper, but I was hoping I was just missing the docs on Vercels site.  

Can you give any insight or point me in the direction of docs in case I missed it.  Just want to get a sense of how much time I should spend on this or if it makes sense to just spin up an S3.  Thanks!

**Comment by SuryaSekhar14** [comment]
You are right, Vercel officially only supports their JS Library to use Vercel Blob Storage and no docs are present for any other language.  I had to reverse engineer the npm package to build the wrapper.

If you are doing a big project like I was, spending a week or two to create your own wrapper is pretty okay, but for a small one, I think spinning up an S3 is the best way to go.
 
 Hope you find this helpful!

**Reply by doodirock** [reply]
Super helpful response. Thanks so much for the feedback.
